BUILDING.md

```diff
@@ -265,16 +265,15 @@ More Developer Tools...`. This step will install `clang`, `clang++`, and
 
 #### Building Node.js
 
-If the path to your build directory contains a space, the build will likely
-fail.
-
```

Comment:
I don't think to remove this is good, I think you could combine two sentences because you additions is explain why likely fail when dir contains a space.